CMakeLists.txt

```diff
@@ -34,13 +34,16 @@ endif()
 
 # Now simply link against gtest or gtest_main as needed. Eg
 
+set(CMAKE_CXX_COMPILER "/usr/local/bin/g++-9")
```

Comment:
if travis require this specific compiler path you can pass it to a cmake command
`cmake .. -DCMAKE_CXX_COMPILER="/usr/local/bin/g++"` for a g++ build